app/views/signals/index.html.erb

```diff
@@ -124,6 +124,7 @@
       <div class="col-sm-6 signals-section__col">
         <div class="signals-section-content signals-section-content--xs-last">
           <div class="signal-example-panels">
+            <div class="b sky-blue">Active! Our beta users are trading with this signal.</div>
```

Comment:
I wonder if it would be better to put this into the component using a prop of active: true ?
I don't know if it will be applied to other signals later, but the placement is a bit confusing.

Reply:
Yeah... it'll almost definitely be used for other signals. I tried that approach and was unsuccessful, but just realized it's likely because I was only running the server rendering Webpack build - will give it another shot with the client side.